// const sum = (a, b) => {
//     if (a && b) {
//         return a / b;
//     }
//     throw new Error('Invalid arguments');
// };
// try {
//     console.log(sum(33));
// } catch (error) {
//     console.log('Error occured');
//     console.log(error);
// }


console.log(Math.floor(Math.random() * 899999 + 100000));